### DISCUSSION TITLE
Package installment suggestion

### DISCUSSION THREAD

**Discussion by Ethan1X** [PaddlePaddle/PaddleSpatial#212] [BaiduKDDCup2022] [answered]
Please install tsfresh in the paddlepadedle environment, thanks.

**Comment by zhanshijinwat** [comment]
@ramber1836 @shaido987 please install tsfresh , thanks

**Reply by zhanshijinwat** [reply]
in paddlepaddle environment

**Reply by shaido987** [reply]
I'm actually one of the participants and not among the organizers =) but @ramber1836 should be able to help you.

**Reply by zhanshijinwat** [reply]
> I'm actually one of the participants and not among the organizers =) but @ramber1836 should be able to help you.

ops, sorry.

**Reply by zhanshijinwat** [reply]
@ramber1836   @zhoujingbo  We make the request before DDL, please help us，thanks.

**Comment by ramber1836** [comment, marked as answer]
This package had already been installed in the paddlepaddle environment in the 2nd phase. 
Please follow the latest update of the Evaluation Manual for the updated list of the runtime installed packages.
